admin/jpbeta-admin-main.php

```diff
@@ -40,6 +40,11 @@
 		list-style-type: none;
 	}
 
+	.j-beta-wrap .card img {
+		max-width: 100%;
+		padding: 1em 1em 1em 0;
```

Comment:
It worked for me without right padding, in Firefox. Which browser are you testing with?

By setting the width and left padding it should fit in the column nicely.